### PR TITLE
 UBERF-8983: Speedup startup in PG adapter

### PR DESCRIPTION
**Pull request by haiodo**
UBERF-8983: Speedup startup in PG adapter

+ Add status support for cloud worker

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NzY1MTkzMGMyOWE3OGFmZmY2NmU3NDIiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIn0.xpJiNK-70Q0tFwdl0veJlz0HTJ2gWTecWGS5MwE5M48">Huly&reg;: <b>UBERF-8985</b></a></sub>

### CODE REVIEW

File: server/postgres/src/utils.ts [hunk 1]
```diff
@@ -71,52 +72,66 @@ export const NumericTypes = [
...
-async function getTableSchema (client: postgres.Sql, domain: string): Promise<void> {
-  const res = await client.unsafe(`SELECT column_name, data_type, is_nullable
-    FROM information_schema.columns
-    WHERE table_name = '${domain}' and table_schema = 'public' ORDER BY ordinal_position ASC;
-  `)
+async function getTableSchema (client: postgres.Sql, domains: string[]): Promise<void> {
+  const res = await client.unsafe(
+    `SELECT column_name, data_type, is_nullable, table_name
+            FROM information_schema.columns
+            WHERE table_name = ANY($1) and table_schema = 'public'
```
Comment (denis-tingaikin) [inline server/postgres/src/utils.ts:112]: [minor] Seems like can be used `in`
```suggestion
            WHERE table_name IN ($1) AND table_schema = 'public'
```

**Review by denis-tingaikin — DISMISSED** [review]
LGTM

File: server/postgres/src/utils.ts (hunk 1 above)
Comment (haiodo) [reply server/postgres/src/utils.ts:112]: @denis-tingaikin any difference between?

**Review by BykhovDenis — DISMISSED** [review]

File: server/postgres/src/utils.ts (hunk 1 above)
Comment (denis-tingaikin) [reply server/postgres/src/utils.ts:112]: No. `in` for cases when we look for a match in a list can be more readable for devs. So comment minor, can be ignored.

Comment (haiodo) [reply server/postgres/src/utils.ts:112]: We should use ANY in this concrete context.

**Review by BykhovDenis — APPROVED** [review]

**Pull request merged**